pandas/_libs/tslibs/offsets.pyx

```diff
@@ -895,13 +895,6 @@ cdef class Tick(SingleConstructorOffset):
 
         raise ApplyTypeError(f"Unhandled type: {type(other).__name__}")
 
-    # --------------------------------------------------------------------
```

Comment:
what were these hitting? or were they at all?

Reply:
I honestly do not know if these were used or not. I ran into the issue that a lot of the `BaseOffset` children were not properly initialized. I ended up initializing `_cache` in every class but noticed they should all do fine using the `__reduce__` in https://github.com/pandas-dev/pandas/blob/ce34c1cee70021792b0c1b0db2c899069bbf0c1d/pandas/_libs/tslibs/offsets.pyx#L728-L737

They are all tested in https://github.com/pandas-dev/pandas/blob/ce34c1cee70021792b0c1b0db2c899069bbf0c1d/pandas/tests/io/test_pickle.py#L170-L177

with the data of https://github.com/pandas-dev/pandas/blob/ce34c1cee70021792b0c1b0db2c899069bbf0c1d/pandas/tests/io/generate_legacy_storage_files.py#L135